tests/php/Controllers/Root/RootController/RootControllerContentsActionTest.php

```diff
@@ -443,4 +443,20 @@ public function testContentsActionIsTabbarStatusEqualHomeWhenIsDevParam()
             'error:アサイン情報としてtabbarStatusはcontentsではない'
         );
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)